### ISSUE TITLE
Error on clone/checkout master branch on windows 

### ISSUE THREAD

**Issue by MugeSo** [kogosoftwarellc/open-api#140]
As a subject, following error message  is displayed when we clone nor checkout master branch on windows.

> error: unable to create file packages/fs-routes/test-dir/users/:id.js: Invalid argument

Because `：` is special character on windows files system.

**Comment by jsdevel** [comment]
@MugeSo does the rest of the project get cloned properly?

**Comment by jsdevel** [comment]
@MugeSo can you try a sparse checkout?  See an example [here](https://stackoverflow.com/questions/26097568/git-pull-error-unable-to-create-file-invalid-argument/34515900#34515900).

I'm guessing that the only ways to solve this are:
1. Remove the offending file from the repo (it may not be absolutely necessary as it's just a test file).
2. Require Windows users to use sparse checkout.

**Comment by MugeSo** [comment]
> does the rest of the project get cloned properly?

Yes, all except `:id.js` exists.
Then `git status` says:
```
deleted:    packages/fs-routes/test-dir/users/:id.js
```

Sparse checkout with following `.git/info/sparse-checkout` file works well for master branch.
```
*
!:*.*
```
However, It still displays warning message when checkout the branch which does not have `:id.js` from master branch.
```
warning: unable to unlink packages/fs-routes/test-dir/users/:id.js: Invalid argument
```

>  I'm guessing that the only ways to solve this are:
>  1. Remove the offending file from the repo (it may not be absolutely necessary as it's just a test file).
>  2. Require Windows users to use sparse checkout.

I like 2 because it's not good to force windows users to set "sparse-checkout".
It will reduce their motivation to contribute.

Even if `:id.js` file is needed for testing, it can be created only for non-windows OS in test code .

**Comment by jsdevel** [comment]
@MugeSo this should be fixed now (see #163)
